maps-compose-utils/src/main/java/com/google/maps/android/compose/clustering/ClusterRenderer.kt

```diff
@@ -168,26 +168,35 @@ internal class ComposeUiClusterRenderer<T : ClusterItem>(
         }
     }
 
-    private fun renderViewToBitmapDescriptor(view: AbstractComposeView): BitmapDescriptor {
+    private fun renderViewToBitmapDescriptor(view: AbstractComposeView?): BitmapDescriptor {
```

Comment:
```suggestion
    private fun renderViewToBitmapDescriptor(view: AbstractComposeView): BitmapDescriptor {
```

Reply:
From your stacktrace, the NPE doesn't happen from this view but it's parent, I believe we have no need to make it nullable.

Reply:
that's true i just realized that thanks